gcc/rust/util/rust-attributes.cc

```diff
@@ -223,27 +223,30 @@ check_doc_attribute (const AST::Attribute &attribute)
       }
     }
 }
-
-static bool
-is_proc_macro_type (const AST::Attribute &attribute)
+tl::optional<std::string>
```

Comment:
Could we return an enum instead of a string here ? Enums are cheap, strings are not. and if I'm reading this correctly string values are known statically.